debt-token/src/lib.rs

```diff
@@ -33,18 +33,7 @@ impl DebtTokenTrait for DebtToken {
     /// Panics if the contract has already been initialized.
     /// Panics if name or symbol is empty
     ///
-    fn initialize(
-        e: Env,
-        decimal: u32,
-        name: Bytes,
-        symbol: Bytes,
-        pool: Address,
-        underlying_asset: Address,
-    ) {
-        if decimal > u8::MAX.into() {
-            panic!("debt-token: decimal must fit in a u8");
-        }
-
+    fn initialize(e: Env, name: Bytes, symbol: Bytes, pool: Address, underlying_asset: Address) {
```

Comment:
Since you're removing `decimal`, could you also remove the mention of it in the documentation above?

`/// - decimal - The number of decimal places for the token.`